### PR TITLE
Adding PackageVersion parameter to NugetPack task so that nuspec version can be overriden

### PR DESCRIPTION
**Pull request by joperezr**
Adding PackageVersion parameter to NugetPack task so that nuspec version can be overriden

### CODE REVIEW

**Comment by joperezr** [comment]
cc @ericstj @weshaggard

**Comment by joperezr** [comment]
This work is needed to fix #430

File: src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs [hunk 1]
```diff
@@ -79,6 +86,16 @@ public override bool Execute()
...
                     }
 
+                    // Overriding the Version from the Metadata if one gets passed in.
+                    if (!string.IsNullOrEmpty(PackageVersion))
+                    {
+                        NuGetVersion overrideVersion;
+                        if (NuGetVersion.TryParse(PackageVersion,out overrideVersion))
+                        {
+                            builder.Version = overrideVersion;
+                        }
```
Comment (mellinoe) [inline src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs]: Should we give an error if TryParse fails?

Comment (ericstj) [inline src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs]: :+1:

**Comment by ericstj** [comment]
lgtm with @mellinoe's suggestion

**Comment by joperezr** [comment]
@dotnet-bot test this please

**Pull request merged**